### PR TITLE
Readme: Changed config file from json to yaml

### PR DESCRIPTION
**Pull request by krealyt**

### CODE REVIEW

**Comment by krealyt** [comment]
@pionix-juraed Please also check if this is still the correct place to put those user config files.

File: config/user-config/README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # User provided configuration
-You can add user provided configuration files in this folder that can augment a loaded config of the same name.
-For example, if you want to change the ***serial_port*** of the ***yeti_driver*** in the [config-hil.json](../config-hil.json) to a more appropriate value on you machine you can achieve this by adding a [config-hil.json](config-hil.json) file in this user-config folder with the following content:
+You can add user provided configuration files in this folder that can augment
+a loaded config of the same name.
+For example, if you want to change the ***serial_port*** of the
+***yeti_driver*** in the [config-hil.yaml](../config-hil.yaml) to a more
+appropriate value on your machine you can achieve this by adding a
+*config-hil.yaml* file in this user-config folder with the following content:
```
Comment (valentin-dimov) [inline config/user-config/README.md:7]: `config-hil.yaml` will probably be removed very soon. Could we use a different example here?

Comment (krealyt) [reply config/user-config/README.md:7]: @valentin-dimov Could you make a very simple example? Or who could?

Comment (valentin-dimov) [reply config/user-config/README.md:7]: Maybe we could suggest changing the `evse_id` of `connector_1` in `config-sil.yaml` to `DE*YOURCOMPANY*E12345*1`?

Comment (krealyt) [reply config/user-config/README.md]: @valentin-dimov Changed as you suggested.

**Review by valentin-dimov — APPROVED** [review]
lgtm!

**Pull request merged**